iosdevapp/Model.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Model.swift
```

Comment:
At the moment you only have two small models. (suggestion: rename the file to `Models`)

If the number of models grows, you should think about putting each of them in a separate file.